R/test-package.r

```diff
@@ -53,12 +53,17 @@ test_package <- function(package, filter = NULL, reporter = "summary", ...) {
 }
 
 
-run_tests <- function(package, test_path, filter, reporter, ...)
-{
+run_tests <- function(package, test_path, filter, reporter, ...) {
   reporter <- find_reporter(reporter)
   env <- test_pkg_env(package)
   res <- with_top_env(env, {
-    test_dir(test_path, reporter = reporter, env = env, filter = filter, ...)
+    results <- test_dir(test_path, reporter = reporter, env = env, filter = filter, ...)
+
+    # TODO: Parse output and append success / failure to results
```

Comment:
Nothing :( You get test output, but no reported failure or anything of that form.

If necessary, we can either write a parser for Catch test output that can then translate results as needed, but I wasn't sure exactly what would need to be done here.